httpserver-rs/src/lib.rs

```diff
@@ -230,7 +229,19 @@ impl HttpServerCore {
                                 http::StatusCode::INTERNAL_SERVER_ERROR
                             }
                         };
-                        *http_response.status_mut() = status;
+                        let http_builder = http::Response::builder()
+                        .status(status);
+                        let http_builder = if let Some(cache_control_header) = arc_inner.settings.cache_control.as_ref(){
+                            let mut builder = http_builder;
+                            for val in cache_control_header{
+                                builder = builder.header("Cache-Control",val)
```

Comment:
Just want to double-check that this appends vs overwrites if there are multiple

Reply:
How does that translate to the response over HTTP? Can a header appear multiple times, or do these need to be concatenated?

Reply:
From what I know, these need to be comma separated instead of individual values. @thomastaylor312 @lachieh may know better.

Reply:
yah, comma separated joined in single header:

`Cache-Control: max-age=0, private, must-revalidate`